app/views/dishes/_form.html.haml

```diff
@@ -0,0 +1,36 @@
+-#-# frozen_string_literal: true
+-#= simple_form_for(@dish) do |f|
+-#  = f.error_notification
+-#  = f.error_notification message: f.object.errors[:base].to_sentence if f.object.errors[:base].present?
+-#
+-#  .form-inputs
+-#    = f.input :name
+-#    = f.input :price
+-#
+-#  .form-actions
+-#    = f.button :submit
+%form
+  .row.mb-3
+    %label.col-sm-2.col-form-label{:for => "@Name"} Name
+    .col-sm-10
+      %input#inputEmail3.form-control{:type => "name"}/
+  .row.mb-3
+    %label.col-sm-2.col-form-label{:for => "Price"} Price
+    .col-sm-10
+      %input#inputPassword3.form-control{:type => "price"}/
+  %fieldset.row.mb-3
+    %legend.col-form-label.col-sm-2.pt-0 Radios
+    .col-sm-10
+      .form-check
+        %input#gridRadios1.form-check-input{:checked => "checked", :name => "gridRadios", :type => "radio", :value => "main_dish"}/
+        %label.form-check-label{:for => "gridRadios1"}
+          Main dish
+      .form-check
+        %input#gridRadios2.form-check-input{:name => "gridRadios", :type => "radio", :value => "soup"}/
+        %label.form-check-label{:for => "gridRadios2"}
+          Soup
+      .form-check
+        %input#gridRadios2.form-check-input{:name => "gridRadios", :type => "radio", :value => "beverage"}/
+        %label.form-check-label{:for => "gridRadios2"}
```

Comment:
gridRadios3, but what do you think provide elements id which describe their value